### ISSUE TITLE
Workflow: Moving some API Response annotations to the class

### ISSUE THREAD

**Issue by jdcmsd** [dotCMS/core#30985]
### Parent Issue

https://github.com/dotCMS/core/issues/28600

### Task

This is sort of a test run to see how if it's an improvement to inherit some baseline HTTP responses from the class when they're constant across all member methods/operations.

Basic behaviors:

- `@ApiReponse` annotation values fed to the class via `@ApiResponses` are present in all class methods.
- When a method uses an `@ApiResponse` annotation with the same code as the class, it overrides for just that method.
- If a method uses an `@ApiResponse` code NOT present in the class, it will not affect any of the others; it still inherits from the class, but now has an extra response code.

In other words, it cleans up the code a bit and removes some redundancy to find response codes that are **present for all methods** and move them to the class in their most common value, then override the less common forms. So, in the case of Workflow, that's 401, 403, and 500.

If there are response codes that are not used by all methods (like 404) it's better to specify them at the method level; I'm not sure if there's a way to override subtractively.

### Proposed Objective

Documentation

### Proposed Priority

Priority 4 - Trivial

### Acceptance Criteria

Move those three response codes into the class.
